test/new-e2e/tests/installer/docker.go

```diff
@@ -79,7 +86,8 @@ sudo systemctl start docker
 // launchJavaDockerContainer launches a small Java HTTP server in a docker container
 // and make a call to it
 func launchJavaDockerContainer(t *testing.T, host *components.RemoteHost) {
-	host.MustExecute(`sudo docker run -d -p8887:8888 baptistefoy702/message-server:latest`)
+
+	host.MustExecute(`sudo docker run -d -p8887:8888 669783387624.dkr.ecr.us-east-1.amazonaws.com/dockerhub/baptistefoy702/message-server:latest`)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**